pkg/plugin/plugin.go

```diff
@@ -60,6 +61,7 @@ var _ framework.PreFilterPlugin = (*AutoscaleEnforcer)(nil)
 var _ framework.PostFilterPlugin = (*AutoscaleEnforcer)(nil)
 var _ framework.FilterPlugin = (*AutoscaleEnforcer)(nil)
 var _ framework.ScorePlugin = (*AutoscaleEnforcer)(nil)
+var _ framework.ScoreExtensions = (*AutoscaleEnforcer)(nil)
```

Comment:
I think this is no longer necessary now? because we're returning `*AutoscaleEnforcer` from `ScoreExtensions()`